Dockerfile

```diff
@@ -1,28 +1,28 @@
-FROM openshift/origin-base
-
-ENV APP_HOME=/opt/app/src \
-    GOPATH=/go \
-    PATH="${PATH}:${GOPATH}/bin:/usr/local/bin" \
-    INSTALL_PKGS="golang make git"
+# first stage
+FROM openshift/origin-release:golang-1.10 AS builder
 
-RUN yum install -y ${INSTALL_PKGS} --setopt=tsflags=nodocs
+ENV GOPATH=/go
```

Comment:
this should already be set in the builder image, and FWIW, downstream includes /go as well as something scl-specific. I don't know for sure if this will cause any issues in the downstream compile. if there's not a specific need for this I'd leave it out.

I'm also curious why all your source goes in /tmp below instead of /go.

Reply:
In the current incarnation, this allows you to build the binary outside of the image.  Not sure how this would play out otherwise

Reply:
The build depends on the /tmp path? *shrug* seems odd but if it works for you...